hiba-ca.sh

```diff
@@ -316,6 +436,32 @@ if [ "$sign" = 1 ]; then
 	fi
 fi
 
+# Check revoke required command line parameters
+if [ "$revoke" = 1 ]; then
+	# the certificate serial is required.
+	if [ -z "$serial" -a -z "$hiba" ]; then
+		usage "missing -z <revocation spec> or -H grant for revocation."
+	elif [ -n "$serial" -a -n "$hiba" ]; then
+		usage "only one of -z <serial> or -H grant allowed at once for revocation."
+	fi
+fi
+
+# Check log required command line parameters
+if [ "$manage" = 1 ]; then
+	# the cut off must by a number of days.
+	if [ "$((days + 0))" -eq 0 ]; then
+		usage "-N must be a integer representing the number of days to display."
+	fi
+fi
+
+# Check cleanup required command line parameters
+if [ "$cleanup" = 1 ]; then
+	# the cut off must by a number of days.
+	if [ "$((days + 0))" -eq 0 ]; then
```

Comment:
Can't seem to remember with I used the math trick. Removing.

For negative numbers, this is actually fine as it will translate to a date in the future and not clean up anything.